stdlib/public/Platform/glibc.android.modulemap.in

```diff
@@ -0,0 +1,373 @@
+//===--- module.map -------------------------------------------------------===//
```

Comment:
Please factor out the duplication in the new module map.  gyb seems to be a good tool for the job.